README.md

```diff
@@ -5,37 +5,32 @@
 An application to provide HTTP API from NineChronicles RPC server.
 
 
-<!-- FIXME: I'm not sure what's better between Dependencies and Installation
-     about this section -->
+## Quick Start
 
-## Dependencies
-
-It imports [planetarium/nekoyume-unity] repository's code as git submodule 
-because it must use RPC client created from code shared with NineChronicles RPC server.
- 
+Checkout source code:
 ```bash
-$ git clone --recurse-submodules git@github.com:<username>/<repository>
+$ git clone --recurse-submodules git@github.com:planetarium/NineChronicles.HttpGateway.git
 ```
 
-[planetarium/nekoyume-unity]: https://github.com/planetarium/nekoyume-unity
-
-
-## Build
-
+Build from source:
 ```bash
 $ dotnet build
 ```
 
-
-## Run
-
+Running developer builds:
 ```bash 
-$ dotnet run -- /rpcServerHost=<string> /rpcServerPort=<short>
-
 $ dotnet run -- /rpcServerHost=aafd1af9c5cf111ea824802399f8ed0e-1178879563.ap-northeast-2.elb.amazonaws.com /rpcServerPort=31234
```

Comment:
Then it is weird that above command successfuly runned from my machine without any error message.
Anyway, I would restore README.md as before except invalid submodule URL since this topic is not purpose of this PR.

Reply:
It will run successfully because it uses it lazily. If the gRPC server was dead and you call the API, you will see the error message in the console. I mean that it must be second if it should be deleted.